Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1

```diff
@@ -133,16 +134,48 @@ function Install-EdFiOdsWebApi {
             WebApiFeatures = @{
                 BearerTokenTimeoutMinutes="23"
                 ExcludedExtensionSources="GrandBend"
-                FeatureIsEnabled=@{
-                    changeQueries = $true
-                    openApiMetadata = $false
-                    composites = $false
-                    profiles = $false
-                    identityManagement = $false
-                    extensions = $false
-                    ownershipBasedAuthorization = $true
-                    uniqueIdValidation = $true
+                Features= @{
+                "OpenApiMetadata"=@{
+                  Name= "OpenApiMetadata"
+                  IsEnabled= $true
                 }
+                "AggregateDependencies"=@{
+                  Name= "AggregateDependencies"
+                  IsEnabled= $true
+                }
+                "TokenInfo"=@{
+                    Name= "TokenInfo"
+                    IsEnabled= $true
+                  }
+                  "Extensions"=@{
+                    Name= "Extensions"
+                    IsEnabled= $true
+                  }
+                  "Composites"=@{
+                    Name= "Composites"
+                    IsEnabled= $true
+                  }
+                  "Profiles"=@{
+                    Name= "Profiles"
+                    IsEnabled= $true
+                  }
+                  "ChangeQueries"=@{
+                    Name= "ChangeQueries"
+                    IsEnabled= $false
+                  }
+                  "IdentityManagement"=@{
+                    Name= "IdentityManagement"
+                    IsEnabled= $false
+                  }
+                  "OwnershipBasedAuthorization"=@{
+                    Name= "OwnershipBasedAuthorization"
+                    IsEnabled= $false
+                  }
+                  "UniqueIdValidation"=@{
+                    Name= "UniqueIdValidation"
+                    IsEnabled= $false
+                  }
+            }
```

Comment:
Here's the result if I use what you have here:
```json
{
  "ApiSettings": {
    "Mode": "SharedInstance",
    "EncryptSecrets": false,
    "ExcludedExtensions": [],
    "Engine": "SqlServer",
    "OdsTokens": [],
    "Features": [
      {
        "Name": "OpenApiMetadata",
        "IsEnabled": true
      },
      {
        "Name": "AggregateDependencies",
        "IsEnabled": true
      },
      {
        "Name": "TokenInfo",
        "IsEnabled": true
      },
      {
        "Name": "Extensions",
        "IsEnabled": true
      },
      {
        "Name": "Composites",
        "IsEnabled": true
      },
      {
        "Name": "Profiles",
        "IsEnabled": false
      },
      {
        "Name": "ChangeQueries",
        "IsEnabled": false
      },
      {
        "Name": "IdentityManagement",
        "IsEnabled": false
      },
      {
        "Name": "OwnershipBasedAuthorization",
        "IsEnabled": false
      },
      {
        "Name": "UniqueIdValidation",
        "IsEnabled": false
      }
    ],
    "UseReverseProxyHeaders": false
  },
  "ExcludedExtensionSources": "GrandBend",
  "Plugin": {
    "Folder": "../../../Plugin",
    "Scripts": []
  },
  "AllowedHosts": "*",
  "ConnectionStrings": {
    "EdFi_Ods": "server=localhost;database=EdFi_Ods;Integrated Security=true;Application Name=Ed-Fi ODS WebApi",
    "EdFi_Security": "server=localhost;database=EdFi_Security;Integrated Security=true;Application Name=Ed-Fi ODS WebApi",
    "EdFi_Admin": "server=localhost;database=EdFi_Admin;Integrated Security=true;Application Name=Ed-Fi ODS WebApi",
    "EdFi_master": ""
  },
  "Features": {
    "IdentityManagement": {
      "IsEnabled": false,
      "Name": "IdentityManagement"
    },
    "Composites": {
      "IsEnabled": true,
      "Name": "Composites"
    },
    "ChangeQueries": {
      "IsEnabled": false,
      "Name": "ChangeQueries"
    },
    "OwnershipBasedAuthorization": {
      "IsEnabled": false,
      "Name": "OwnershipBasedAuthorization"
    },
    "TokenInfo": {
      "IsEnabled": true,
      "Name": "TokenInfo"
    },
    "AggregateDependencies": {
      "IsEnabled": true,
      "Name": "AggregateDependencies"
    },
    "Profiles": {
      "IsEnabled": true,
      "Name": "Profiles"
    },
    "Extensions": {
      "IsEnabled": true,
      "Name": "Extensions"
    },
    "OpenApiMetadata": {
      "IsEnabled": true,
      "Name": "OpenApiMetadata"
    },
    "UniqueIdValidation": {
      "IsEnabled": false,
      "Name": "UniqueIdValidation"
    }
  },
  "BearerTokenTimeoutMinutes": "23",
  "Logging": {
    "LogLevel": {
      "Default": "Debug",
      "Microsoft": "Warning"
    }
  }
}
```
This creates another `Features` object in the JSON. We need it to override the existing `Features` object. In JSON `Features` is an `array` of `objects` with `Name` and `IsEnabled` properties. So in Powershell this would be an `array` of `hashtables`. If we don't get the format right the website won't know what to do with these settings.